src/SFML/Audio/SoundFileReaderFlac.cpp

```diff
@@ -40,7 +40,7 @@
 
 namespace
 {
-FLAC__StreamDecoderReadStatus streamRead(const FLAC__StreamDecoder*, FLAC__byte buffer[], std::size_t* bytes, void* clientData)
+FLAC__StreamDecoderReadStatus streamRead(const FLAC__StreamDecoder*, FLAC__byte* buffer, std::size_t* bytes, void* clientData)
```

Comment:
I’d rather retain the official function signature. Yes, it’s equivalent, but I don’t agree with changing it, just to make our tooling happy.